apps/examples/testcase/ta_tc/messaging/utc/utc_messaging_recv.c

```diff
@@ -76,6 +76,8 @@ static void block_sender(int argc, FAR char *argv[])
 
 	data.msglen = strlen(TC_BLOCK_MSG) + 1;
 	data.priority = 100;
+	data.msg = NULL;
```

Comment:
Why is this necessary?

Reply:
It is not necessary.
I will remove this line.